tools/msvs/install_tools/install_tools.bat

```diff
@@ -47,7 +47,23 @@ echo script is at your own risk. Please read the Chocolatey's legal terms of use
 echo and the Boxstarter project license as well as how the community repository
 echo for Chocolatey.org is maintained.
 echo.
-echo You can close this window to stop now.
+pause
+
+cls
+echo !!!!!WARNING!!!!!
+echo -----------------
+echo Use of Boxstarter may reboot your box automatically multiple times. When
```

Comment:
Nit-picking a bit, but "reboot your box" is rather colloquial. "reboot your computer" or something like that is probably preferable.

Reply:
Sounds good to me!